src/index.ts

```diff
@@ -61,7 +65,7 @@ export const createInstance = (config: Config): Instance => {
     getDomainById: subgraphClient.getDomainById,
     getDomainsByName: subgraphClient.getDomainsByName,
     getDomainsByOwner: subgraphClient.getDomainsByOwner,
-    getSubdomainsById: subgraphClient.getSubdomainsById,
```

Comment:
@Remscar @JamesEarle  Is this being consumed already by the dapp? If so passivity is likely important. @JamesEarle you may have to perform data transformation on the dstore api response to get it into the format that is expected.

Reply:
We should make using the datastore api toggleable